config/htaccess.yaml

```diff
@@ -23,7 +23,7 @@
 ### Giza requires this file, even if no redirects
 ### More as a safety -- in case, we remove redirects during deploys
 #######################################################################
-### Redirects to put /v2.0/ in the URL                              ###
+### Redirects to put /current in the URL                                                                                    ###
```

Comment:
I believe this will work.  I'm now second guessing why v2.0 was used instead of current in the first place.  @kay-kim any feedback on this?

Ditto for the conf-sitemap.xml